package.json

```diff
@@ -39,50 +38,50 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
-    "bunsen-core": "0.18.1",
-    "ember-browserify": "1.1.13",
-    "ember-bunsen-core": "0.12.1",
-    "ember-cli": "^2.7.0",
+    "bunsen-core": "~0.18.2",
+    "ember-browserify": "^1.1.13",
+    "ember-bunsen-core": "~0.12.2",
```

Comment:
Please pin `ember-bunsen-core` and `bunsen-core` to exact versions. I don't want to risk breaking changes in those.